node-js-basics-101/node_modules/wordwrap/test/break.js

```diff
@@ -0,0 +1,30 @@
+var assert = require('assert');
+var wordwrap = require('../');
+
+exports.hard = function () {
+    var s = 'Assert from {"type":"equal","ok":false,"found":1,"wanted":2,'
+        + '"stack":[],"id":"b7ddcd4c409de8799542a74d1a04689b",'
```

Comment:
__[jshint]__
* Misleading line break before '+'; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->